Dodanie pliku z linkami o KIFie

przemekli opened the pull request.

przemekli committed:
Create KIF.md

Adds page with links about KIF

przemekli committed:
Update Readme.md

Dodanie informacji i odnośnika o nowym pliku (na temat frameworku KIF)

pwicherski commented on Materialy/Narzedzia/KIF.md:
```diff
@@ -0,0 +1,49 @@
+
+* https://github.com/kif-framework/KIF
+    * Strona główna projektu
+* https://groups.google.com/forum/#!forum/kif-framework
```
Grupa dyskusyjna to nie repozytorium :) Sugeruję inną nazwę sekcji zamiast "Repozytorium" lub inny podział.

pwicherski commented on Materialy/Narzedzia/KIF.md:
Niepotrzebna spacja.

pwicherski commented on Materialy/Narzedzia/KIF.md:
Niepotrzebna spacja. W kilku miejscach poniżej także :)

przemekli committed:
Update KIF.md

Przenosi link do grupy dyskusyjnej z sekcji "Repozytorium" do sekcji "Materiały"

pwicherski commented:
Rozdziel proszę źródła na następujące sekcje:
- Materiały od autorów
- Materiały - różne
- Blogi
- Materiały wideo
- Kursy

Jeżeli oczywiście któryś materiał będzie pasował do danej sekcji :)

Możesz wziąć za przykład plik Appium:
https://github.com/pwicherski/TestowanieOprogramowania/blob/master/Materialy/Narzedzia/Appium.md

przemekli committed:
Update KIF.md

Usuwa niepotrzebne spacje

przemekli committed:
Poprawki po uwagach

Wnosi poprawki po komentarzach do pliku

przemekli commented:
Poprawki naniesione. Brakuje szablonu i stąd tak wyszło ;)

pwicherski commented on Materialy/Narzedzia/KIF.md:
Niepotrzebna spacja.

pwicherski commented on Materialy/Narzedzia/KIF.md:
Niepotrzebna spacja.

pwicherski commented on Materialy/Narzedzia/KIF.md:
Duplikat źrodła. To samo źródło w "Blogi"

przemekli committed:
Update KIF.md

Poprawki po review

przemekli committed:
Update KIF.md

Usuwa zbędną spację
